docs/spec/overview.md

```diff
@@ -86,17 +86,25 @@ multiple _Subscriptions_.
 
 See [Kind: Channel](spec.md#kind-channel).
 
-## Provisioner
+## ClusterProvisioner
```

Comment:
I intentionally didn't rename the `Provisioner` topic to `ClusterProvisioner`. While ClusterProvisioner is the current CRD name, we will also likely have a Provisioner CRD in the future. The generic discussion is around provisioners.